code/modules/hydroponics/unique_plant_genes.dm

```diff
@@ -309,7 +309,7 @@
 	/// Spawned mob's melee damage = this multiplier * seed potency.
 	var/mob_melee_multiplier = 1
 	/// Spawned mob's move delay = this multiplier * seed potency.
-	var/mob_speed_multiplier = 1
+	var/mob_speed_multiplier = 0.5
```

Comment:
The children of this datum override this, 1's just a default value
(`/datum/plant_gene/trait/mob_transformation/tomato` and `/datum/plant_gene/trait/mob_transformation/shroom`)

Reply:
Really good catch, thank you